tools/packaging/common/istio-start.sh

```diff
@@ -21,7 +21,7 @@
 set -e
 
 # Match pilot/docker/Dockerfile.proxyv2
-export ISTIO_META_ISTIO_VERSION="1.3.0"
+export ISTIO_META_ISTIO_VERSION="1.6.0"
```

Comment:
Do we have this string in other places as well?  It would be great to have a single source of truth.

Reply:
Its normally based by the release builder. We should have a single source of truth, but we don't have a great way to do that for this I think. I created https://github.com/istio/istio/issues/20566 to track improvements

Reply:
sg ... thanks!